### PR TITLE
pueue 0.11.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> cargo install --locked --root /usr/local/Cellar/pueue/0.11.0 --path .
error: found a virtual manifest at `/private/tmp/pueue-20210118-23247-1ho4lu0/pueue-pueue-v0.11.0/Cargo.toml` instead of a package manifest
```

File: Formula/pueue.rb [hunk 1]
```diff
@@ -72,7 +73,10 @@ def plist
       Process.kill("TERM", pid)
     end
 
-    assert_match "Pueue daemon #{version}", shell_output("#{bin}/pueued --version")
-    assert_match "Pueue client #{version}", shell_output("#{bin}/pueue --version")
+    # bug report about the version in the release artifact
+    # hopefully revert the change in next release
+    # https://github.com/Nukesor/pueue/issues/169
```
Comment (SMillerDev) [inline Formula/pueue.rb:78]: Can't we get a real test instead?

Comment (bayandin) [reply Formula/pueue.rb:78]: A version with a fix has been released: https://github.com/Nukesor/pueue/releases/tag/pueue-v0.11.1, but we definitely need a better test in addition to this one.

Comment (chenrui333) [reply Formula/pueue.rb:78]: I will give a shot.

Comment (chenrui333) [reply Formula/pueue.rb]: I think it should be good, let me know your thought.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**